### ISSUE TITLE
'Login error, please correct data' error when attempting to log in with correct information?

### ISSUE THREAD

**Issue by Feldherren** [phubbard/Meltdown#9]
I'm unsure whether it can't connect for some reason, or whether it's failing to parse the URL, due to the error it's giving. I am reasonably sure I've given it the correct information, however.

URL is: www.feldherren.[host].com/fever
(Host removed for some limited attempt at privacy.)
I'm very certain I'll have entered the correct information at least once, between all attempts, so presumably something else is wrong here.

**Comment by phubbard** [comment]
My apologies; that could be either a username/password error, or an SSL problem. I need to add better error messages here.

Please double-check for trailing spaces from autocorrect. 

Do you have a self-signed SSL cert? Meltdown uses the systems' cert system, so if the browser complains then Meltdown will silently fail.

Accepted as bug, will try and fix. Thanks!

**Comment by Feldherren** [comment]
In this case I'm just using free hosting somewhere, so I'm not entirely certain how to check. I can tell you that Fever itself works more-or-less normally in Chrome, though. There aren't any trailing spaces in any field, or leading for that matter.

**Comment by phubbard** [comment]
OK. I'll add some logging and see what we can find. Can you tell me the Android version, or any other variables you think could affect it? (Proxies?)

**Comment by Feldherren** [comment]
Certainly. I'm using 4.2.2 on a Nexus 7. Haven't set up anything to do with proxies for the primary connection I'm testing this with; I have to note that I have no connectivity problems other than being unable to use the app, and can easily access my copy of Fever through a browser if I need to - don't have any problems accessing it that way. I have also tested it with another connection and had no joy there, either. Sorry I'm not being terribly helpful, but at my end all the app is doing is telling me something is wrong when I try to log in.

**Comment by camillo** [comment]
Hello,

My installation runs on a subdomain. The ssl key is for the main domain, so I am not able to provide a correct cert. It would be great if there could be a checkbox to accept all certs. I would really like to use ssl, even if the cert is invalid.
